src/migmose/mig/segmentgrouphierarchies.py

```diff
@@ -0,0 +1,53 @@
+"""
+contains class for trees consisting of segments of mig tables
+"""
+
+import json
+from pathlib import Path
+from typing import Any, Optional
+
+from loguru import logger
+from maus.edifact import EdifactFormat
+from pydantic import BaseModel, Field
+
+from migmose.mig.reducednestednachrichtenstruktur import ReducedNestedNachrichtenstruktur
+
+
+class SegmentGroupHierarchy(BaseModel):
+    """Contains the model for a segment group hierarchy used by the MAUS library."""
+
+    opening_segment: Optional[str] = None
+    segment_group: Optional[str] = None
+    sub_hierarchy: list[Optional["SegmentGroupHierarchy"]] = Field(default_factory=list)
+
+    @classmethod
+    def create_segmentgroup_hierarchy(
+        cls, reduced_nested_nachrichtenstruktur: ReducedNestedNachrichtenstruktur
+    ) -> "SegmentGroupHierarchy":
+        """init Segmentrgroup Hierarchy"""
+        segment_group: Optional[str] = None
+        opening_segment: Optional[str] = None
+        if reduced_nested_nachrichtenstruktur.header_linie is not None:
+            segment_group = reduced_nested_nachrichtenstruktur.header_linie.bezeichnung
+        if reduced_nested_nachrichtenstruktur.segmente and reduced_nested_nachrichtenstruktur.segmente[0] is not None:
```

Comment:
```suggestion
        if any(reduced_nested_nachrichtenstruktur.segmente) and reduced_nested_nachrichtenstruktur.segmente[0] is not None:
```
ich finde truethy/falsy ausdrücke nicht immer leicht zu lesen.